components/python/pyrsistent/manifests/generic-manifest.p5m

```diff
@@ -0,0 +1,68 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+
+# Copyright 2021 <contributor>
+set name=pkg.fmri value=pkg:/$(COMPONENT_FMRI)@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.summary value="$(COMPONENT_SUMMARY)"
+set name=info.classification value="$(COMPONENT_CLASSIFICATION)"
+set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
+set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
+set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
+license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
+file path=usr/lib/python2.7/vendor-packages/_pyrsistent_version.py
+file path=usr/lib/python2.7/vendor-packages/pyrsistent-$(COMPONENT_VERSION)-py2.7.egg-info/PKG-INFO
+file path=usr/lib/python2.7/vendor-packages/pyrsistent-$(COMPONENT_VERSION)-py2.7.egg-info/SOURCES.txt
+file path=usr/lib/python2.7/vendor-packages/pyrsistent-$(COMPONENT_VERSION)-py2.7.egg-info/dependency_links.txt
+file path=usr/lib/python2.7/vendor-packages/pyrsistent-$(COMPONENT_VERSION)-py2.7.egg-info/top_level.txt
+file path=usr/lib/python2.7/vendor-packages/pyrsistent/__init__.py
+file path=usr/lib/python2.7/vendor-packages/pyrsistent/__init__.pyi
+file path=usr/lib/python2.7/vendor-packages/pyrsistent/_checked_types.py
+file path=usr/lib/python2.7/vendor-packages/pyrsistent/_field_common.py
+file path=usr/lib/python2.7/vendor-packages/pyrsistent/_helpers.py
+file path=usr/lib/python2.7/vendor-packages/pyrsistent/_immutable.py
+file path=usr/lib/python2.7/vendor-packages/pyrsistent/_pbag.py
+file path=usr/lib/python2.7/vendor-packages/pyrsistent/_pclass.py
+file path=usr/lib/python2.7/vendor-packages/pyrsistent/_pdeque.py
+file path=usr/lib/python2.7/vendor-packages/pyrsistent/_plist.py
+file path=usr/lib/python2.7/vendor-packages/pyrsistent/_pmap.py
+file path=usr/lib/python2.7/vendor-packages/pyrsistent/_precord.py
+file path=usr/lib/python2.7/vendor-packages/pyrsistent/_pset.py
+file path=usr/lib/python2.7/vendor-packages/pyrsistent/_pvector.py
+file path=usr/lib/python2.7/vendor-packages/pyrsistent/_toolz.py
+file path=usr/lib/python2.7/vendor-packages/pyrsistent/_transformations.py
+file path=usr/lib/python2.7/vendor-packages/pyrsistent/py.typed
+file path=usr/lib/python2.7/vendor-packages/pyrsistent/typing.py
+file path=usr/lib/python2.7/vendor-packages/pyrsistent/typing.pyi
+file path=usr/lib/python$(PYVER)/vendor-packages/_pyrsistent_version.py
```

Comment:
The python2.7 entries look odd. Did you run gmake clean after removing the python 2.7 version? It looks like your build folder was dirty when you run gmake sample-manifest.